allennlp/tests/data/dataset_readers/dataset_utils/span_utils_test.py

```diff
@@ -45,6 +45,39 @@ def test_bio_tags_to_spans_ignores_specified_tags(self):
         spans = span_utils.bio_tags_to_spans(tag_sequence, ["ARG1", "V"])
         assert set(spans) == {("ARG2", (6, 7)), ("ARG2", (9, 9))}
 
+    def test_iob1_tags_to_spans_extracts_correct_spans_without_labels(self):
+        tag_sequence = ["I", "B", "I", "O", "B", "I", "B", "B"]
+        spans = span_utils.iob1_tags_to_spans(tag_sequence)
+        assert set(spans) == {("", (0, 0)), ("", (1, 2)), ("", (4, 5)), ("", (6, 6)), ("", (7, 7))}
```

Comment:
I'm not a huge fan of empty strings as `None` values - Do things still work in metrics/CRF if we make them `None` explicitly?

Reply:
I made them empty strings because the final output metrics would be stored under the keys `B-`, `I-`, `O-`, etc., which feels more natural to me than having them under `B-None`, `I-None`, `O-None`.
alternatively, we can special case this `None` case and store the keys as `B`, `I`, `O`...do you think that's preferable?

Reply:
but i also haven't looked at the CRF model to see if it breaks with unlabeled tags...I think the only thing that might be affected is the `allowed_transitions` function?

Reply:
I'm merging this for now, but happy to change this in a future PR if you want it.